src/msg.c

```diff
@@ -1517,9 +1377,8 @@ int msg_start(struct settings *settings)
 	}
 
 	device_id_list = l_queue_new();
-	start_to =  l_timeout_create_ms(1, start_timeout, settings, NULL);
 
-	return (start_to ? 0 : -ENOMEM);
+	cloud_list_devices();
```

Comment:
Since we don't have a l_timeout to keep retring we should verify the return of this function and try again if had some error

Reply:
:eyes: You should also return 0 if success otherwise it will stop all the services that you started

Reply:
Ok